src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -85,3 +85,306 @@ Feature: GCS source - Verification of GCS to GCS Additional Tests successful
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Validate the data transferred from GCS Source to GCS Sink with Expected avro file and target data in GCS bucket
+
+  @GCS_CSV @GCS_SINK_TEST @EXISTING_GCS_CONNECTION
+  Scenario: To verify data is getting transferred from GCS Source to GCS Sink using test Schema Detection On Single File with connection
```

Comment:
nit: `To verify data is getting transferred from GCS Source to GCS Sink using Schema Detection On Single File with connection`
`test` seems unnecessary here.
This comment applies to all similar scenarios.

Reply:
As per the ITN Class in Single File and Multiple file with regex (useConnection is true) and in Multiple File with different schema (useConnection is not used). So, We are removing a particular tests which has not used useConnection in Multiple file with different schema. If you are suggesting that to remove unnecessary tests, then we can remove those tests.

1)Removing a scenario for multiple file with different schema with connection(True).
2)Removing a scenario for multiple file with regex with connection(False).
3)Removing a scenario for single file with connection(False).

Reply:
I just suggested to remove `test` word from the scenario name not the whole scenario.

Sorry for the confusion.

Reply:
Removed test **word** from the scenario.